keps/sig-storage/1847-autoremove-statefulset-pvcs/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Auto remove PVCs created by StatefulSet
+kep-number: 1847
+authors:  
+  - "@kk-src"
+  - "@dsu-igeek"
+  - "@mattcary"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-apps
+status: provisional
```

Comment:
We can update to implementable once comments are addressed

Reply:
Just to reiterate, this will need to be implementable